Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -110,6 +142,30 @@ void ABenchmarkGymGameMode::Tick(float DeltaSeconds)
 				}
 			}
 		}
+
+		for (int i = AIControlledPlayers.Num() - 1; i >= 0; i--)
+		{
```

Comment:
Can't this logic just be simplified to if controller and character are good, call `ClientSetBlackboardAILocations`, then once whole array is processed, empty it?